sigle/src/modules/editor/EditorHeader.tsx

```diff
@@ -80,7 +80,7 @@ export const EditorHeader = ({
     >
       <Flex gap={{ '@initial': '5', '@md': '10' }} align="center">
         <Link href="/" passHref>
-          <IconButton as="a" aria-label="Go back to the dashboard">
+          <IconButton size="sm" as="a" aria-label="Go back to the dashboard">
```

Comment:
It's pretty strange that all the buttons across the app are of size sm, the default is md so I guess one would expect that most of them are actually of this size, or the default is not the right one?

Reply:
yeah I mentioned this one to Quentin and he said that most of the buttons will be updated to become md—but as we're not there yet I guess it makes sense to change the default variant to sm?

Reply:
Okay, if long term we are going to have md let's keep it as a default 👍